community_images/yourls/official/image.yml

```diff
@@ -0,0 +1,39 @@
+name: yourls
+official_name: YOURLS
+official_website: https://yourls.org/
+source_image_provider: YOURLS
+source_image_repo: docker.io/_/yourls
+source_image_repo_link: https://hub.docker.com/_/yourls
+source_image_readme: https://github.com/docker-library/docs/blob/master/yourls/README.md
+rf_docker_link: rapidfort/yourls
+image_workflow_name: yourls_official
+github_location: yourls/official
+report_url: https://frontrow.rapidfort.com/app/community/imageinfo/docker.io%2Fbitnami%2Fyourls
```

Comment:
Please fix the link. bitnami should be library

Reply:
Rectifying it.